### ISSUE TITLE
Documentation: Removing duplicate entries

### ISSUE THREAD

**Issue by tdilo** [openchargemap/ocm-system#144]
I was unable to figure out in a reasonable amount of time how to remove a duplicate entry from the database. The editor interface itself only allows to mark the charging station as decommissioned, which does not remove the entry from the map. It is possible to post a comment to mark the entry as a duplicate, but then nothing further seems to happen (is it up to a country editor to search for those comments and act?).

The list of charging stations nearby would be a good place to show a checkbox to mark other entries as duplicates, when they are reasonably close together.

**Comment by webprofusion-chrisc** [comment]
Duplicates are a tricky one currently, only editors can set the Submission status from published to Delisted - Duplicate so it does rely on them noticing your intention currently. 

The danger of delisting a duplicate is that you remove a more accurate listing (or one that has checkin-ins or photos) leaving a possibly less useful listing still published, which is why it's restricted to Editors because they need to use their judgement. Other examples include sites with multiple network operators in close proximity, which we required to be added as two sites (with similar position).

We have no UI currently for editors to search by comment type (such as the appropriately titled 'Location is a duplicate') and the long term plan would be to have such UI in the app/map rather than the web UI, so they can edit on the go but it may be that we have to build this UI in the website instead.

**Comment by tdilo** [comment]
Good point about multiple network operators. Another case may be a set of slow chargers next to fast chargers, which may have the same operator but a slightly different location and user base. It could be useful to keep those separated as well.

If entries were merged instead of delisting one set of data outright, the decision would not be as difficult - but it still seems like an editorial task to decide in case of conflicting data. Marking duplicates by ID in one of the entries of the set of duplicates would allow to hide the others from the map. The system could simply show check-ins and photos from the duplicates in the main entry. The duplicate entries itself would still be accessible through the main entry chosen by the user in form of a list of duplicates. At some point in time an editor could resolve this merge by hand and actually delist the duplicates, transferring applicable check-ins and photos.

Building UI for editors to list all user comments requiring action (duplicate entries, charge point does not exist anymore) sounds like the way to go in either case.
